en/02_Developer_Guides/15_Customising_the_Admin_Interface/08_ReactJS_Redux_and_GraphQL.md

```diff
@@ -255,6 +254,9 @@ const addLoggingMiddleware = (next) => (error) => {
 If you've created a module using React, it's a good idea to afford other developers an
 API to enhance those components, forms, and state. To do that, simply register them with `Injector`.
 
+> [!WARNING]
+> Because of the unique structure of the `form` middleware, you cannot register new services to `Injector.form`.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.